cmd/crates/soroban-test/tests/it/integration/bindings.rs

```diff
@@ -0,0 +1,35 @@
+use soroban_test::{TestEnv, LOCAL_NETWORK_PASSPHRASE};
+use super::util::deploy_swap;
+
+pub const OUTPUT_DIR: &str = "./bindings-output";
```

Comment:
```suggestion
const OUTPUT_DIR: &str = "./bindings-output";
```
Is this used outside this file?